### ISSUE TITLE
Remove MPL-2.0 Features

### ISSUE THREAD

**Issue by Archdoog** [Rallista/maplibre-compose-playground#59]
In an effort to get this library accepted to @maplibre alongside https://github.com/maplibre/swiftui-dsl. We need to remove all MPL-2.0 code. This includes anything in the following folders:

- https://github.com/Rallista/maplibre-compose-playground/tree/main/compose/src/main/java/com/maplibre/compose/ramani
- Some of https://github.com/Rallista/maplibre-compose-playground/tree/main/compose/src/main/java/com/maplibre/compose/symbols

And anything else with:

```kt
/*
 * This file is part of ramani-maps.
 *
 * Copyright (c) 2023 Roman Bapst & Jonas Vautherin.
 *
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at https://mozilla.org/MPL/2.0/.
 */
 ```

While some of this may be challenging, we've already diverged the MapView quite a bit to get more dynamic feature sets in there. 

As far as how we achieve this, it may be easiest to start a new repository with a fresh start on the MapView that takes the learnings of this repo + the SwiftUI-DSL. Then pull in the basics like the MapViewCamera that were contributed to only to this repo.

Thoughts? This is an interesting situation where this repository aims to parallel the swift-dsl in features and API design. So it makes sense to get it to a BSD-3 or MIT license.

**Comment by ianthetechie** [comment]
Yeah, I agree. The benefits of having an API that's at least spiritually aligned (of course there will be some idiomatic differences) are huge for developers that need to work on both platforms.

> As far as how we achieve this, it may be easiest to start a new repository with a fresh start on the MapView that takes the learnings of this repo + the SwiftUI-DSL. Then pull in the basics like the MapViewCamera that were contributed to only to this repo.

Agreed. My understanding of MPL-2.0 is that it's roughly per source code file. And since we have git history, we can verify before each PR or commit that we're not bringing in something that was incompatibly licensed. Then we'll have to rework the rest, but you've already been redoing a lot of the architecture anyways, so hopefully not too much of a lift...
